src/Header.jsx

```diff
@@ -0,0 +1,52 @@
+import { useState } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faBars } from "@fortawesome/free-solid-svg-icons";
+import { faCircleUser } from "@fortawesome/free-solid-svg-icons";
+import NavBar from "./NavBar";
+import "./Header.css";
+
+function Header() {
+  const [showMenu, setShowMenu] = useState(false);
+  const toggleMenu = () => setShowMenu((s) => !s);
+
+  const handleClose = () => setShowMenu(false);
+
+  return (
+    <header>
+      {!showMenu && (
+        <aside className="menu">
```

Comment:
Can we change the order of className="login" and className="header-menu-items" so that login is on the right and the menu items are on the left.

Reply:
This has been done